### DISCUSSION TITLE
OS Password entering doesnt work and it just freezes your screen (1.2.3 and windows 11)

### DISCUSSION THREAD

**Discussion by justasillyguy** [rustdesk/rustdesk#6426] [General]
### Bug Description

If the other computer is on the lock screen and you try to enter the OS password using the button, it doesn't work, your screen just freezes, doesn't work on pencil or on actual button.

### How to Reproduce

1. Connect to a remote (Windows 10) computer
2. Lock it using the Start menu
3. Try and use the OS password option

### Expected Behavior

Should let you log in

### Operating system(s) on local side and remote side

Windows 11 -> Windows 11

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/150978007/06236a67-07b6-4a86-9a47-3b130e6290bb)

![image](https://github.com/rustdesk/rustdesk/assets/150978007/7ec6e21d-46ee-436d-85fe-37bde08fa130)

![image](https://github.com/rustdesk/rustdesk/assets/150978007/4c4d47af-0be6-4d75-83f3-3040d053fc77)

**Comment by justasillyguy** [comment]
By the way this is my first time reporting an issue so sorry if I've done anything bad

**Comment by 21pages** [comment]
Only works if the controlled side is installed or elevated in tests.

**Comment by justasillyguy** [comment]
> Only works if the controlled side is installed or elevated in tests.

It has been installed, may need to elevate though
